hoomd/md/nlist.py

```diff
@@ -95,6 +101,14 @@ def __init__(self, buffer, exclusions, rebuild_check_delay, check_dist):
         params["exclusions"] = exclusions
         self._param_dict.update(params)
 
+        self._mesh = validate_mesh(mesh)
+
+    def _attach(self):
+        if self._mesh:
```

Comment:
```suggestion
        if self._mesh is not None:
```
While your code is considered Pythonic, I find this explicit check to be clearer than relying on the truthiness of the variable.